picoquictest/crypto_api_test.h

```diff
@@ -0,0 +1,38 @@
+/*
+* Copyright (c) 2020, The PicoQuic Authors.
+* All rights reserved.
```

Comment:
Same remark as above on copyright to Private Octopus.

Reply:
👍 I will add the files like this right now, but we can change it as soon as we defined what you prefer